### DISCUSSION TITLE
Request for authorization to use Thonny in educational material

### DISCUSSION THREAD

**Discussion by TheJeffah** [thonny/thonny#3403] [General]
Hello everyone,

I am creating a book aimed at Python beginners, with the goal of providing an accessible and practical introduction to programming. I plan to use Thonny as the tool throughout the book due to its simplicity and ease of use. I decided not to include other IDEs because I believe Thonny is the ideal choice for those starting to learn Python.

I would like to know if I can include Thonny as the recommended IDE in my book, which will be published in digital format on Amazon.

Thanks for your attention, and I am available to provide more details about the project or answer any questions.

Best regards,
The Jeffah

**Comment by aivarannamaa** [comment]
You are welcome to!

**Comment by TheJeffah** [comment]
Great, @aivarannamaa ! Thanks a lot for your support. I'll keep in touch !
